tests/big_bird/test_modeling_big_bird.py

```diff
@@ -581,7 +581,7 @@ def test_fast_integration(self):
             self.assertTrue(
                 torch.allclose(
                     hidden_states[0, 0, :5],
-                    torch.tensor([1.4943, 0.0928, 0.8254, -0.2816, -0.9788], device=torch_device),
+                    torch.tensor([1.4825, 0.0774, 0.8226, -0.2962, -0.9593], device=torch_device),
```

Comment:
updated the expected values, since the config is changed to `block_size=8` and the model behave differently now (run with "block_sparse")